app.js

```diff
@@ -0,0 +1,140 @@
+function Menu(name) {
+    this.name = name;
+}
+
+Menu.prototype.calculatePrice = function() {
+    return this.name.price;
+}
+
+Menu.prototype.calculateCalories = function() {
+    return this.name.calories;
+}
+
+function Hamburger(name, ...stuffing) { 
+    Menu.call(this, name);
+    this.stuffing = stuffing;
+} 
+
+Hamburger.SIZE_SMALL = { type: 'small', price: 50, calories: 20 };
+Hamburger.SIZE_LARGE = { type: 'large', price: 100, calories: 40 };
+Hamburger.STUFFING_CHEESE = { type: 'cheese', price: 10, calories: 20 };
+Hamburger.STUFFING_SALAD = { type: 'salad', price: 20, calories: 5 };
+Hamburger.STUFFING_POTATO = { type: 'potato', price: 15, calories: 10 };
+
+Hamburger.prototype = Object.create(Menu.prototype);
+
+Hamburger.prototype.calculatePrice = function () {
+    let count_stuff = 0;
```

Comment:
Considering you use ES5 in this HW, it is better to use var for this task for consistency.